src/firefly/java/edu/caltech/ipac/firefly/server/util/QueryUtil.java

```diff
@@ -40,10 +47,20 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 import java.util.stream.Collectors;
 
-import static edu.caltech.ipac.firefly.core.background.BackgroundStatus.*;
+import static edu.caltech.ipac.firefly.core.background.BackgroundStatus.ACTIVE_REQUEST_CNT;
+import static edu.caltech.ipac.firefly.core.background.BackgroundStatus.ITEMS;
+import static edu.caltech.ipac.firefly.core.background.BackgroundStatus.MESSAGE_CNT;
+import static edu.caltech.ipac.firefly.core.background.BackgroundStatus.PACKAGE_CNT;
+import static edu.caltech.ipac.firefly.core.background.BackgroundStatus.RESPONSE_CNT;
+import static edu.caltech.ipac.firefly.core.background.BackgroundStatus.TOTAL_BYTES;
```

Comment:
We should formally define this in Firefly style guide.  If not, with different IntelliJ setup,  it'll unnecessarily changes code.
IMO, with modern editor, explicitly defining all imports is redundant.  As a matter of fact, it collapses them anyways to avoid cluttering.  But, I am okay with either way as long as it's consistent.

Reply:
yea, that change was accidental. We could decide.